packages/web-api/package.json

```diff
@@ -54,7 +54,8 @@
     "form-data": "^2.5.0",
     "is-stream": "^1.1.0",
     "p-queue": "^6.6.1",
-    "p-retry": "^4.0.0"
+    "p-retry": "^4.0.0",
+    "is-electron": "^2.2.0"
```

Comment:
It's fine to add this one (MIT license - well-maintained) for now. If we enable passing a constructor argument to disable user-agent header (function or boolean) in the future, we don't need to have this external dependency.